### PR TITLE
FRB Status + KKO

### PR DESCRIPTION
**Pull request by profxj**
## Description of the Change

- Calculate `frb_status` from the DB
- Integrate KKO sources
- Build out FFFF-PZ views

## Checklist

- [ x] HTML code change
- [ ] Added package dependency
- [ ] Added data
- [ x] Changed django model
- [ ] Documentation change

### CODE REVIEW

File: YSE_App/frb_status.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    'RunDeepPATH', # Needs PATH run on deeper (typically private) imaging
+    'ImagePending', # Pending deeper imaging with an FRBFollowUp
+        # FRB appears in FRBFollowUpRequest with mode='image'
+    'SpectrumPending', # Pending spectroscopy with an FRBFollowUp
+        # FRB appears in FRBFollowUpRequest with mode='longslit','mask'
+    'GoodSpectrum', # Observed with spectroscopy successfully
+    'TooFaint', # Host is too faint for spectroscopy
+        # r-magnitude (or equivalent; we use the PATH band) of the top host candidate
+        #   is fainter than the maximum(mr_max) for the sample/surveys
+    'AmbiguousHost',  # Host is consideed too ambiguous for further follow-up
```
Comment (tarraneh) [inline YSE_App/frb_status.py:34]: consideed > considered

File: YSE_App/frb_status.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        # P(U|x) is set
+        # Deep imaging must exist.  The list of telescope+intrument is below
+        # P(U|x) > P_Ux_max
+    'Redshift', # Redshift measured
+]
+
+# List of telescope+instruments that are considered Deep
+deep_telinstr = []
+
+# List of frb_rags where one should run a public PATH
```
Comment (tarraneh) [inline YSE_App/frb_status.py:47]: frb_rags >> frb_tags

**Review by tarraneh — APPROVED** [review]

File: YSE_App/frb_status.py (hunk 1 above)
Comment (profxj) [reply YSE_App/frb_status.py:34]: fixed

File: YSE_App/frb_status.py (hunk 2 above)
Comment (profxj) [reply YSE_App/frb_status.py:47]: fixed

**Review by lordrick94 — APPROVED** [review]

**Pull request merged**